nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/config/ElasticSearchIndexConfig.java

```diff
@@ -210,4 +219,33 @@ public void merge(final ElasticSearchIndexConfig other) {
         }
     }
 
+    // @since 9.3
+    public boolean hasExplicitWriteIndex() {
+        return writeAlias != null && ! writeAlias.isEmpty();
```

Comment:
No space after `!` (several occurrences). Anyway you should use `StringUtils.isEmpty` instead.